theme.json

```diff
@@ -342,6 +342,23 @@
 				}
 			},
 			"core/site-title": {
+				"elements": {
+					"link": {
+						":hover": {
+							"typography": {
+								"textDecoration": "underline"
```

Comment:
Should the underline be dashed or solid? 🤔

Reply:
@beafialho Could you confirm which underlines should be dashed and which should be solid?

Reply:
Does this help @mikachan?

<img width="1262" alt="Captura de ecrã 2022-08-22, às 15 29 46" src="https://user-images.githubusercontent.com/65220155/185946701-f2d53efc-0bc0-4848-9e7a-950ed3f8d4af.png">